### PR TITLE
fix: fallback endpoint connection

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Pick the fallback endpoint one by one (serialize) instead of parallel fetching as devops team requested
* Removed deprecated code 

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

### CODE REVIEW

File: src/config/api/polkadot/connectApi.ts [hunk 1]
```diff
@@ -52,29 +52,72 @@ const loadAccounts = async (api: ApiPromise) => {
   );
 };
 
-const fallBackConnection = async (networkIdx: number): Promise<void> => {
-  await Promise.race([
-    providerEndpoints[networkIdx].endpoints.map(async (it) => {
-      try {
+const fallbackConnection = async ({
```
Comment (hoonsubin) [inline src/config/api/polkadot/connectApi.ts:55]: please add a comment to the function's behavior (and other functions too)

File: src/boot/api.ts [hunk 2]
```diff
@@ -38,11 +39,11 @@ export default boot(async ({ store }) => {
 
   const networkIdx = computed(() => store.getters['general/networkIdx']);
 
-  const defaultEndpoint = providerEndpoints[networkIdx.value].endpoints[0].endpoint;
+  const randomEndpoint = getRandomFromArray(providerEndpoints[networkIdx.value].endpoints).endpoint;
```
Comment (hoonsubin) [inline src/boot/api.ts:42]: https://user-images.githubusercontent.com/40356749/166207222-b158509a-2277-4727-9970-ec6973f4c68f.mov

When the user switches the network, the endpoint should be randomized, but when I try to switch from one network to another, the modal will revert my option back to the first option.

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/boot/api.ts (hunk 2 above)
Comment (impelcrypto) [reply src/boot/api.ts:42]: Fixed.

Demo:
https://www.loom.com/share/ed6a8232ea9f465dab28f9facd3cdfb0

Code:
https://github.com/AstarNetwork/astar-apps/pull/375/commits/15eaff184ed6494592c50a796f063fcafb5d8466
https://github.com/AstarNetwork/astar-apps/pull/375/commits/0560a87967d0baafb0f6cea7d94bbdf5033276dd

File: src/config/api/polkadot/connectApi.ts (hunk 1 above)
Comment (impelcrypto) [reply src/config/api/polkadot/connectApi.ts:58]: Fixed.

https://github.com/AstarNetwork/astar-apps/pull/375/commits/3390ac6fcab9fb9a0e54172f2a1b9753a0cf0969
https://github.com/AstarNetwork/astar-apps/pull/375/commits/0560a87967d0baafb0f6cea7d94bbdf5033276dd

**Review by hoonsubin — APPROVED** [review]
LGTM

**Review by sirius651 — APPROVED** [review]

**Pull request merged**